src/gc/gcpriv.h

```diff
@@ -1116,9 +1116,9 @@ struct no_gc_region_info
 };
 
 //class definition of the internal class
-#if defined(GC_PROFILING) || defined(FEATURE_EVENT_TRACE)
+#if defined(GC_PROFILING) && defined(FEATURE_EVENT_TRACE)
```

Comment:
Is this change really correct? I can see the original form of the condition used at about 50 places in CoreCLR sources.